app/views/layouts/_sidebar.html.slim

```diff
@@ -1,3 +1,4 @@
-div.sidebar.sidebar-left.sidebar-animate#site-navigation-sidebar class=sidebar_classes
-  ul.nav.navbar-stacked
-    = yield
+div class=sidebar_classes
+  nav.navbar-default.sidebar role='navigation'
+    div.collapse.navbar-collapse#site-navigation-sidebar
```

Comment:
This can't have an ID because e.g. in Course there are two of these, because sidebar_items are called twice.